starlette/middleware/sessions.py

```diff
@@ -19,9 +15,10 @@ def __init__(
         max_age: int = 14 * 24 * 60 * 60,  # 14 days, in seconds
         same_site: str = "lax",
         https_only: bool = False,
+        backend: SessionBackend = None,
     ) -> None:
         self.app = app
-        self.signer = itsdangerous.TimestampSigner(str(secret_key))
+        self.backend = backend or CookieBackend(secret_key, max_age)
```

Comment:
Having `secret_key` be a required parameter of the middleware for the default cookieBackend makes for some pretty strange ergonomics when using other backends, even though I understand it's meant to remain compatible with the existing middleware's signature.

Reply:
Yes, I also would like to drop it.